assets/cubyz/biomes/forest.json

```diff
@@ -22,7 +22,7 @@
 			"leaves" : "cubyz:oak_leaves",
 			"log" : "cubyz:oak_log",
 			"top" : "cubyz:oak_top",
-			"chance" : 0.03,
+			"chance" : 0.5,
```

Comment:
Is it intentional that you move from 3:1 oak:birch to 10:1?

Reply:
No, I just didn't do the math. fixed it.